.github/workflows/configurebb.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+if [ "${BUILDBUDDY_API_KEY:-}" ]; then
```

Comment:
could this be more similar to how rules_js and rules_ts do it, for consistency?

Reply:
veering away from that since I also want to the configure the `--remote_cache` here on all CI steps so we're get faster build times even when RBE is not on

Reply:
and the GHA ci config yaml has multiple matrices so went with bash script to avoid duplication